python/example_code/lookoutvision/csv_to_manifest.py

```diff
@@ -0,0 +1,202 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier:  Apache-2.0
+"""
+Purpose
+Shows how to create an Amazon Lookout for Vision manifest file from a CSV file.
+The CSV file format is image location,anomaly classification (normal or anomaly)
+For example:
+s3://s3bucket/circuitboard/train/anomaly/train_11.jpg,anomaly
+s3://s3bucket/circuitboard/train/normal/train_1.jpg,normal
+
+If necessary, use the bucket argument to specify the S3 bucket folder for the images.
+More information: https://docs.aws.amazon.com/lookout-for-vision/latest/developer-guide/ex-csv-manifest.html
+"""
+
+# snippet-start:[python.example_code.lookoutvision.Scenario_CSVtoManifest]
+
+from datetime import datetime, timezone
+import argparse
+import logging
+import csv
+import os
+import json
+
+logger = logging.getLogger(__name__)
+
+
+def check_errors(csv_file):
+    """
+    Checks for duplicate images and incorrect classifications in a CSV file.
+    If duplicate images or invalid anomaly assignments are found, an errors CSV file
+    and deduplicated CSV file are created. Only the first
+    occurence of a duplicate is recorded. Other duplicates are recorded in the errors file.
+    :param csv_file: The source CSV file
+    :return: True if errors or duplicates are found, otherwise false.
+    """
+
+    logger.info("Checking %s.", csv_file)
+
+    errors_found = False
+    errors_file = f"{os.path.splitext(csv_file)[0]}_errors.csv"
+    deduplicated_file = f"{os.path.splitext(csv_file)[0]}_deduplicated.csv"
+
+    with open(csv_file, 'r') as input_file,\
+            open(deduplicated_file, 'w') as dedup,\
+            open(errors_file, 'w') as errors:
```

Comment:
I didn't know you could chain together multiple `open` statements like this. Nice!